packages/gestalt/src/Masonry.js

```diff
@@ -72,11 +85,14 @@ const VIRTUAL_BUFFER_FACTOR = 0.7;
 
 const layoutNumberToCssDimension = n => (n !== Infinity ? n : undefined);
 
+/**
+ * TODO this should be renamed to MasronyBeta or something else so it is clear
```

Comment:
Lets not leave TODO's in the code - either file an issue or fix it :)

Reply:
ok. will file an issue ticket!